packaging/windows/OpenRA.nsi

```diff
@@ -129,13 +129,13 @@ Section "Game" GAME
 	File "${SRCDIR}\RedAlert.exe"
 	File "${SRCDIR}\TiberianDawn.exe"
 	File "${SRCDIR}\Dune2000.exe"
-	File "${SRCDIR}\OpenRA.Game.exe"
-	File "${SRCDIR}\OpenRA.Game.exe.config"
-	File "${SRCDIR}\OpenRA.Utility.exe"
-	File "${SRCDIR}\OpenRA.Server.exe"
+	File "${SRCDIR}\OpenRA.exe"
+	File "${SRCDIR}\OpenRA.Game.dll"
```

Comment:
"${SRCDIR}\Game.dll"

Reply:
we also need `File "${SRCDIR}\OpenRA.Game.dll"` this is the OpenRA.Game project.

Reply:
Ah yeah, I was confused by the other game exe/dll references that should actually have been OpenRA.dll